backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/rest/controller/SpecificationController.kt

```diff
@@ -37,15 +29,22 @@ class SpecificationController @Autowired constructor(
         return specificationDtoMapper.fromDomain(specification)
     }
 
-    @PutMapping("/{specificationId}")
-    fun updateSpecification(@RequestBody specificationFileDto: SpecificationFileDto, @PathVariable specificationId: String): SpecificationDto {
+    @PutMapping("/{urlPathId}")
```

Comment:
That variable name makes it sound like it's the ID for a urlPath. How about "specificationIdFromPath"?